src/sass_context.cpp

```diff
@@ -375,6 +375,11 @@ extern "C" {
     return options;
   }
 
+  const char* ADDCALL sass_copy_c_string (const char* input_string)
```

Comment:
Just checked and it should be `char*` since ie `sass_context_take_error_message` also return a `char*` without const, and the have the same purpose. Seeing this a `sass_free_c_string` actually should be there to free these.